### DISCUSSION TITLE
[Code Reviews] ➡️ Add Button for Next Comment

### DISCUSSION THREAD

**Discussion by cpholguera** [community/community#10614] [Mobile] [answered]
It's very uncomfortable to make a review/respond to review comments on mobile and I end up needing my laptop to do a "proper review".

But it shouldn't necessarily be the case, here are some (hopefully easy to implement) suggestions to improve the experience:

- after commiting suggested changes remember the scroll position and return to it.
- and / or offer a button on the top to go to the next (also previous?) comment. That would be extremely convenient.

Thank you!

**Comment by taki-on** [comment, marked as answer]
Hi @cpholguera, thank you for your feedbacks and awesome suggestions! I will forward your ideas to the team, and we will discuss further those ideas. Thank you again!

**Reply by cpholguera** [reply]
Thanks to you for considering it and making such a great job!
